libsql/src/hrana/cursor.rs

```diff
@@ -0,0 +1,369 @@
+// https://github.com/tursodatabase/libsql/blob/main/docs/HRANA_3_SPEC.md#cursor-entries
+
+use crate::hrana::proto::{Batch, BatchResult, Col, StmtResult, Value};
+use crate::hrana::{CursorResponseError, HranaError, Result, Row};
+use bytes::Bytes;
+use futures::{ready, Future, Stream, StreamExt};
```

Comment:
This code is going to be used in Cloudflare, which doesn't have tokio as a dependency.

Reply:
tokio_steam does not bring in tokio and is 100% compatible. If its not then there is likely something very wrong. FYI this tokio crate does pull in `futures_core::Steam` but that is it, the rest of the handlers are maintained by the tokio team and I trust that much more than the futures crate.

Reply:
@LucioFranco correct me if I'm wrong, but it looks like `tokio-stream` uses tokio as a dependency: https://github.com/tokio-rs/tokio/blob/master/tokio-stream/Cargo.toml

Reply:
Yes, because it just leverages the sync features and no runtime stuff so its not actually tied to tokio, none of the sync features are tied to tokio and/or mean that they can be used in any context that is async (any runtime etc).